src/adm/tiles.rs

```diff
@@ -164,7 +164,9 @@ pub async fn get_tiles(
             .await
             .map_err(|e| {
                 // ADM servers are down, or improperly configured
-                HandlerErrorKind::AdmServerError(e.to_string())
+                let mut err: HandlerError = HandlerErrorKind::AdmServerError().into();
```

Comment:
We might want to keep the string for now and instead break down the type of error (later). Most of what we're seeing so far is [err.is_timeout()](https://docs.rs/reqwest/0.11.3/reqwest/struct.Error.html#method.is_timeout), so we'd probably start there -- but probably be handy to know if it's something else before we fully break it all down.

Reply:
Oh but the string's obviously sent as an extra (I'm not sure I noticed this 🤷‍♂️ ), so we're not losing it, this works for me.

We could map specific e.g. `err.is_timeout()` errors to their own individual error if we wanted later.